### PR TITLE
Use space-separated paths instead of comma-separated

### PR DESCRIPTION
**Pull request by am11**
#### Problem

1. in Windows as well as in Unix-like operating systems, if the file path contains comma, we cannot reliably use `--files` option.
2. we are currently incompatible with shell globbing expansion; without using additional tools as workaround (such as https://github.com/dotnet/format/issues/9#issuecomment-541097833).

#### Solution

Change the semantics of `--include` or `--files` and `--exclude` options from `comma-separated` to `space-separated`.

Now we can do things like `dotnet-format --files {src,test}/**/*.cs` or
`git ls-files :/{src,tests}/*.cs | xargs dotnet-format --files`.

#### Additional change

Change the output message to include full path to file, so we can distinguish the clashing files.

```diff
- dotnet-format --files ./fileA.cs,"a sub directory/fileA.cs"
+ dotnet-format --files ./fileA.cs "a sub directory/fileA.cs"

-  Formatted code file 'fileA.cs'.
-  Formatted code file 'fileA.cs'.
+  Formatted code file '/path/fileA.cs'.
+  Formatted code file '/path/a sub directory/fileA.cs'.
```

### CODE REVIEW

**Comment by JoeRobich** [comment]
I like this change. I wish it had worked this way from the start. Could you update the  readme docs that use these options to match the new syntax as part of this PR.

**Comment by am11** [comment]
@JoeRobich, updated the readme to reflect changes. :)

File: src/Program.cs [hunk 1]
```diff
@@ -33,11 +33,11 @@ private static async Task<int> Main(string[] args)
                 {
                     Argument = new Argument<string>(() => null)
                 },
-                new Option(new[] { "--include", "--files" }, Resources.A_comma_separated_list_of_relative_file_or_folder_paths_to_include_in_formatting_All_files_are_formatted_if_empty)
+                new Option(new[] { "--include", "--files" }, Resources.A_list_of_relative_file_or_folder_paths_to_include_in_formatting_All_files_are_formatted_if_empty)
                 {
                     Argument = new Argument<string>(() => null)
```
Comment (am11) [inline src/Program.cs]: @JoeRobich, this was changed to `string[]` and also the `--exclude` case below.

Comment (JoeRobich) [reply src/Program.cs]: That is what I get for trying to fix merge conflicts using the GH editor. sorry

Comment (am11) [reply src/Program.cs]: No problem, thanks for the merge. I think API should error out in this case. Opened https://github.com/dotnet/command-line-api/issues/787 for discussion.

**Pull request merged**